pom.xml

```diff
@@ -295,7 +295,7 @@
         <servlet.version>3.1.0</servlet.version>
         <joda-time.version>2.3</joda-time.version>
         <thrift.version>0.12.0</thrift.version>
-        <junit.jupiter.version>5.3.2</junit.jupiter.version>
+        <junit.jupiter.version>5.5.0-M1</junit.jupiter.version>
```

Comment:
This is needed for the `@ValueSource` with booleans used in storm-hdfs-blobstore tests.